docs/partners/rest-api/README.md

```diff
@@ -0,0 +1,17 @@
+# Partners REST API Endpoint Documentation
+
+An alternative to manually using the [Jetpack Partner Portal](#) is to query the REST API endpoints that it uses.
```

Comment:
Will you be adding a link here?

Reply:
Yes, once it launches. Right now the portal is A8C-only.